hysr/defaults.py

```diff
@@ -1,20 +1,46 @@
-import typing, pathlib
+import typing, pathlib, inspect
+import pam_models
+import pam_interface
+import pam_mujoco
+
+
+def _get_default(f: typing.Callable[..., typing.Any], arg_name: str) -> typing.Any:
+    # e.g.
+    # def f(a: str="example"): pass
+    # _get_default(f,"a") # returns "example.
+    return inspect.signature(f).parameters[arg_name].default
 
 
 class Defaults:
 
     # scene
-    position_robot: typing.Tuple[float, float, float] = (0.435, 0.1175, -0.0025)
-    orientation_robot: str = "-1 0 0 0 -1 0"
-    position_table: typing.Tuple[float, float, float] = (0.1, 0, -0.44)
-    orientation_table: str = "-1 0 0 0 -1 0"
+    position_robot: typing.Tuple[float, float, float] = _get_default(
+        pam_mujoco.MujocoRobot.__init__, "position"
+    )
+    orientation_robot: str = _get_default(
+        pam_mujoco.MujocoRobot.__init__, "orientation"
+    )
+    position_table: typing.Tuple[float, float, float] = _get_default(
+        pam_mujoco.MujocoTable.__init__, "position"
+    )
+    orientation_table: str = _get_default(
+        pam_mujoco.MujocoTable.__init__, "orientation"
+    )
 
     # mujoco
     mujoco_period: float = 0.002  # seconds
 
     # pam robot config file
-    pam_config: typing.Dict[str, pathlib.Path] = {
-        "real": pathlib.Path("/opt/mpi-is/pam_interface/pam.json"),
-        "sim": pathlib.Path("/opt/mpi-is/pam_interface/pam_sim.json"),
+    pam_config: typing.Dict[pam_mujoco.RobotType, typing.Dict["str", pathlib.Path]] = {}
+
+    pam_config[pam_mujoco.RobotType.PAMY1] = {
```

Comment:
I am surprised that this kind of dictionary access is possible in a class definition :D